decidim-core/spec/helpers/decidim/icon_helper_spec.rb

```diff
@@ -5,10 +5,17 @@
 module Decidim
   describe IconHelper do
     describe "#component_icon" do
+      let(:redesign_enabled) { false }
       let(:component) do
         create(:component, manifest_name: :dummy)
       end
 
+      before do
+        # rubocop:disable RSpec/AnyInstance
+        allow_any_instance_of(ActionView::Base).to receive(:redesign_enabled?).and_return(redesign_enabled)
```

Comment:
Could we move this to a shared context that is included in the target spec as this seems to be repeated across multiple specs?